### ISSUE TITLE
Breaks webpack 2's css-loader when publicPath is an http:// url

### ISSUE THREAD

**Issue by crystalline** [jeerbl/webfonts-loader#9]
To reproduce the bug clone https://github.com/crystalline/webfonts-loader/tree/reproduce-loader-bug
```
cd test 
yarn
node run.js
```
This produces such errors:
```
ERROR in ./~/css-loader!./~/webfonts-loader!./octicons.font.js
Module not found: Error: Can't resolve './http:/localhost:8080/9a0b787f9c5f5bf121d882c684e70901-Octicons.eot' in '/home/user/soft/webfonts-loader/test'
 @ ./~/css-loader!./~/webfonts-loader!./octicons.font.js 6:85-164
 @ ./octicons.font.js
 @ ./entry.js
```

**Comment by jeerbl** [comment]
Ok I'll accept this as soon as the test compilation succeeds (respect `semistandard` notation).

**Comment by pista329** [comment]
Hi. Is this actually fixed? 

Whether I use `publicPath` or `dest` with http:// address, webfonts-loader translates this path to e.g. `./http:/path-to-cdn.tld/static/fonts/icons.svg`. Hence the error messages above.

Problem is with line #184. https://github.com/jeerbl/webfonts-loader/blob/master/index.js#L184

If variable `pub` is full url (e.g. http://localhost:8080), `path.join` will produce `http:/localhost:8080`

**Comment by jeerbl** [comment]
@crystalline Did v1.0.0 fix this for you?

**Comment by pista329** [comment]
@jeerbl no, v1.0.0 did not fixed this issue. This problem exists since fontgen package and seems to be reported multiple times.

While `pub` variable might be populated from `publicPath`, `path.join` will replace double dash in `http://cdn-url` with single dash `http:/cdn-url`.

**Comment by jeerbl** [comment]
@pista329 PR welcomed, don't have much time this week.

**Comment by jeerbl** [comment]
Hi @pista329 

Please have a look at your PR. Once you validate this I will release.

Cheers
Jerome

**Comment by jeerbl** [comment]
Fixed in v1.0.1
